### ISSUE TITLE
Adding / Modifying Tags for Testing/A11y 

### ISSUE THREAD

**Issue by rjsheperd** [openforis/collect-earth-online#1132]
### Problem
Certain elements that are click-able are not accessible via keyboard navigation. This prevents both automated browser testing and makes the app less accessible.

### Pages
- [ ] Home Page (Institutions List, Collapse/Expand buttons)
- [ ] Review Institution Page (Nav, Project links)
- [ ] Project Page (Stats dropdown, Collapse/Expand buttons)

### Screenhots
<img width="508" alt="Collect_Earth_Online" src="https://user-images.githubusercontent.com/1829313/110706705-254c1300-81ad-11eb-8521-23af0848692a.png">

<img width="954" alt="Collect_Earth_Online" src="https://user-images.githubusercontent.com/1829313/110706811-457bd200-81ad-11eb-9444-f956a0709f22.png">

<img width="383" alt="Cursor_and_Collect_Earth_Online" src="https://user-images.githubusercontent.com/1829313/110706616-00f03680-81ad-11eb-9820-93c0340a14b3.png">
